helpers/apolloClient.test.js

```diff
@@ -17,7 +17,6 @@ jest.mock('@apollo/client', () => ({
     cache: { restore: jest.fn() }
   }))
 }))
-jest.mock('nodemailer')
```

Comment:
I don't think we should remove mocking mail for unit tests. 

If we want to test the mail, we should only do it explicitly in the integration tests and e2e tests.

Reply:
I didn't remove nodemailer mocking, I made a manual mock under `__mocks__` that is automatically applied.